app.rb

```diff
@@ -1,3 +1,4 @@
+# -*- encoding : utf-8 -*-
```

Comment:
Good to know! Just pushed the fix.
https://github.com/codeforamerica/balance/pull/358

On Tue, Jul 19, 2016 at 8:58 PM, Dave Guarino <notifications@github.com>
wrote:

> In app.rb
> <https://github.com/codeforamerica/balance/pull/359#discussion_r71450623>:
>
> > @@ -1,3 +1,4 @@
> > +# -*- encoding : utf-8 -*-
>
> 👍 It's also necessary because forcing utf-8 actually removes Spanish
> characters we use like ó
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/codeforamerica/balance/pull/359/files/8116419063d69b7d08800c5b4f4d10fd0d005aa1#r71450623>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKntfnhuLkHIEDB16wGxKF3Rg0zLm4QWks5qXXLMgaJpZM4JP_EH>
> .
>



-- 
*Yami Medina*
*Web Developer: JavaScript, Ruby on Rails, HTML, CSS, and more!*
https://www.github.com/yamilethmedina
https://www.linkedin.com/in/yamilethmedina